doc/man7/ossl-guide-migration.pod

```diff
@@ -327,6 +327,15 @@ context and property query and will call an extended version of the key/IV
 derivation function which supports these parameters. This includes
 L<EVP_PBE_CipherInit_ex(3)>, L<EVP_PBE_find_ex(3)> and L<EVP_PBE_scrypt_ex(3)>.
 
+=head4 PKCS#12 KDF vs. FIPS
+
+Unlike in 1.x, due to code restructuring, the PKCS12KDF algorithm used when
```

Comment:
delete _due to code restructuring_.  That's not the reason.

The reason is FIPS has expanded to encompass "compound" algorithms like KDFs since the 1.0.x FIPS Object Module was last validated.